resources/puppetlabs/lein-ezbake/template/pe/ext/build_defaults.yaml

```diff
@@ -6,7 +6,7 @@ packager: 'puppet'
 gpg_key: '7F438280EF8D349F'
 sign_tar: FALSE
 # a space separated list of mock configs
-final_mocks: 'pupent-el6-x86_64 pupent-el7-x86_64 pupent-el8-x86_64 pupent-sles12-x86_64'
+final_mocks: 'pupent-el6-x86_64 pupent-el7-x86_64 pupent-el8-x86_64 pupent-redhatfips-7-x86_64 pupent-sles12-x86_64'
```

Comment:
I know packaging (sort of) uses these values, but I'm not totally sure how/where. The `pupent` thing is something I've only ever seen in build_defaults, so I'm not totally sure if this is formatted correctly or not (e.g. should this be `pupent-redhatfips7-x86_64` (no space between platform and version) to match the rest??), or whether there needs to be some additional setup for this.

Reply:
Yeah I don't really know why they're formatted differently from the foss build_defaults but will update!

Reply:
Legacy from old build infra that we no longer use, but packaging still parses and uses the names. It's from the old mock builds: https://github.com/rpm-software-management/mock/wiki

Reply:
Ah gotcha. Updated.